testsuite/cluster-test/src/main.rs

```diff
@@ -158,16 +180,56 @@ struct ClusterTestRunner {
     thread_pool: threadpool::ThreadPool,
 }
 
+fn parse_host_port(s: &str) -> failure::Result<(String, u32)> {
+    let v = s.split(':').collect::<Vec<&str>>();
+    if v.len() != 2 {
+        return Err(format_err!("Failed to parse {:?} in host:port format", s));
+    }
+    let host = v[0].to_string();
+    let port = v[1].parse::<u32>()?;
+    Ok((host, port))
+}
+
+impl BasicSwarmUtil {
+    pub fn setup(args: &Args) -> Self {
+        if args.peers.is_empty() {
+            panic!("Peers not set in args");
+        }
+        let parsed_peers: Vec<_> = args
+            .peers
+            .iter()
+            .map(|peer| parse_host_port(peer).unwrap())
+            .collect();
+        Self {
+            cluster: Cluster::from_host_port(parsed_peers, &args.mint_file),
+        }
+    }
+
+    pub fn emit_tx(self, accounts_per_client: usize, thread_params: EmitThreadParams) {
+        let mut emitter = TxEmitter::new(&self.cluster);
+        let _job = emitter.start_job(EmitJobRequest {
```

Comment:
Did you check if it works? I am not super familiar with rust threading nuances, do other threads continue to work after main thread terminated?

Reply:
You're right. When the main thread exits, the process exits taking down other threads along with it.

I have added `thread::park();` which blocks the main thread.